### PR TITLE
Fix queryspec bug in GetFromOwnerStrategy class

### PR DESCRIPTION
**Pull request by Pro100proff**
fixed the parametr queryspec bug

### CODE REVIEW

File: crnk-core/src/main/java/io/crnk/core/repository/foward/strategy/GetFromOwnerStrategy.java [hunk 1]
```diff
@@ -49,7 +49,7 @@
 
 			QuerySpec idQuerySpec = new QuerySpec(targetInformation);
```
Comment (remmeier) [inline crnk-core/src/main/java/io/crnk/core/repository/foward/strategy/GetFromOwnerStrategy.java]: I guess idQuerySpec can also be removed then?

Comment (Pro100proff) [reply crnk-core/src/main/java/io/crnk/core/repository/foward/strategy/GetFromOwnerStrategy.java]: Yes, I've forgot remove it

**Comment by Pro100proff** [comment]
Tell me please, when are you going to review my fixes?

**Comment by remmeier** [comment]
thx! sorry for the delay, I overlooked the second commit. next week a 2.9 release will be available, in the meantime it is available in the latest repo.

**Pull request merged**